internal/providers/pluginfw/resources/volume/data_volumes.go

```diff
@@ -35,7 +35,7 @@ type VolumesList struct {
 }
 
 func (d *VolumesDataSource) Metadata(ctx context.Context, req datasource.MetadataRequest, resp *datasource.MetadataResponse) {
-	resp.TypeName = pluginfwcommon.GetDatabricksStagingName(dataSourceName)
+	resp.TypeName = pluginfwcommon.GetDatabricksProductionName(dataSourceName)
```

Comment:
Debatable decision 2 - do we want to roll them all out at once or in a more gradual fashion?

Reply:
As discussed, mentioning for visibility -- we could roll them out individually in different releases (this would help if one resource contains a bug and people can pin the other version but won't help if the resource with bug is the first one that would be released as other following releases will have that resource anyway.

As the effort to do a release is low right after one successful release, we can create separate releases just making other resource use ProductionName to minimise blast radius but usually people don't pin the tf version so they would be using the latest version anyway.

I think both approaches have same value but given the effort is low for staggered releases, I say we can make them default in different release.

Reply:
Updated